src/components/maintenance/maintenance-document-categories-tab.js

```diff
@@ -231,26 +202,46 @@ function MaintenanceDocumentCategoriesTab() {
     );
   };
 
+  const showEmptyState = !allDocumentCategories.length;
+
   return (
     <div className="MaintenanceDocumentCategoriesTab">
-      <div className="MaintenanceDocumentCategoriesTab-controls">
-        <FilterInput
-          size="large"
-          className="MaintenanceDocumentCategoriesTab-filter"
-          value={filter}
-          onChange={handleFilterChange}
-          placeholder={t('filterPlaceholder')}
+      {!!fetchingData && <Spinner /> }
```

Comment:
I decided to introduce the `EmptyState` component, streamlined with the Dashboard, since on this tab data is missing at first and can also be missing later if categories are removed.
At the very least, a more inviting first look 😊

Reply:
Definitely :+1: